src/utils/engine-status.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * The current states of an lnd-engine
+ *
+ * UNKNOWN - Default state
+ * UNAVAILABLE - lnrpc UNAVAILABLE, wallet unlocker rpc UNAVAILABLE
+ * NEEDS_WALLET - Wallet does not exist,	lnrpc UNIMPLEMENTED, genSeeds does not throw
+ * LOCKED - lnrpc UNIMPLEMENTED, genSeeds throws "wallet exists"
+ * UNLOCKED - lnrpc getInfo does not throw
+ * VALIDATED - lnrpc getInfo matches our configuration
+ *
+ * @constant
+ * @type {Object<key, String>}
+ * @default
+ */
+const ENGINE_STATUS = Object.freeze({
```

Comment:
Not a fan of this being defined in utils. Perhaps config, or even at the top level?

Reply:
@treygriffith top level works for me